Expecto/Expecto.fs

```diff
@@ -549,7 +549,8 @@ module Tests =
       |> listTests config.joinWith
       0
     | ArgsRun config ->
-      runTestsWithCancel ct config tests
+      config.filter tests
```

Comment:
Isn't this already done on this line?

https://github.com/haf/expecto/blob/9001f4736fb9c234723fffe3210959ad30f98331/Expecto/Expecto.fs#L518

Reply:
Yes, you're right. This won't make a difference.